src/pudl/metadata/resources/ferc714.py

```diff
@@ -221,15 +221,19 @@
         "create_database_schema": True,
     },
     "core_pudl__assn_ferc714_pudl_respondents": {
-        "description": (
-            "Home table for PUDL derived FERC 714 respondent IDs. These ID's are used to connect "
-            "older CSV data which uses different respondent IDs than the newer XBRL entity IDs. "
-            "These IDs are manually assigned when new FERC 714 data is is integrated, and any "
-            "newly found utilities are added to "
-            "the list with a new ID. "
-            "This table is read in from a CSV stored in the PUDL "
-            "repository: src/pudl/package_data/glue/respondent_id_ferc714.xlsx"
-        ),
+        "description": {
+            "table_type_code": "entity",
+            "additional_summary_text": "PUDL-derived FERC 714 respondent IDs.",
```

Comment:
Due to the `table_type_code` override this will display as
> Entity table containing static information about PUDL-derived FERC 714 respondent IDs.

Even though the table name still says `assn`.  This felt like the least bad option between that and "Association table providing connections between PUDL-derived FERC 714 respondent IDs and nothing else, it's just the one ID."